syntax = "proto3";

package contracts.services.operations;

import "contracts/services/operations/operation_receipt.proto";

message GetOperationReceiptRequest {
  string operation_id = 1;
}

message GetOperationReceiptResponse {
  OperationReceipt receipt = 1;
}